fagiClient/src/test/java/com/fagi/guitests/ConversationTests.java

```diff
@@ -55,10 +57,12 @@ public static void intialize() {
     }
 
     @AfterEach
-    void Dispose() {
+    void dispose() {
         deleteFolder("MyUsername/conversations");
         deleteFolder("MyUsername");
         deleteFolder("conversations");
+        threadPool.printThreads();
```

Comment:
This call should be made after we stop the threads such that we can see if any are not interrupted